src/main/java/seedu/address/model/participant/Participant.java

```diff
@@ -150,23 +156,25 @@ public String toString() {
                 .append("; Address: ")
                 .append(getAddress());
 
+
         Set<Tag> tags = getTags();
         if (!tags.isEmpty()) {
             builder.append("; Tags: ");
             tags.forEach(builder::append);
         }
 
-        if (!nextOfKins.isEmpty()) {
-            builder.append("; NOK: ");
-            nextOfKins.forEach(builder::append);
-        }
-
         Set<Note> notes = getNotes();
         if (!notes.isEmpty()) {
             builder.append("; Notes: ");
             notes.forEach(builder::append);
         }
 
+        ArrayList<NextOfKin> nextOfKins = getNextOfKins();
+        if (!nextOfKins.isEmpty()) {
+            builder.append("; Next Of Kins: ");
+            nextOfKins.forEach(builder::append);
+        }
+
```

Comment:
Should the next of kins be infront of Notes? I think this can be up for discussion. (Like the sequence of content in the String)(Minor issue)

Reply:
I put NOKs as the last field it is really arbitrary since we will be using GUI for showing these data anyway so we will be fetching different data using getters to input them into a different areas in our application. If there were to be any changes in the future this method can be changed accordingly